Projects/App/Sources/Journeys/LoggedInNavigation.swift

```diff
@@ -508,6 +465,49 @@ class LoggedInNavigationViewModel: ObservableObject {
         }
     }
 
+    func handleEditCoInsured() {
+        let contractStore: ContractStore = globalPresentableStoreContainer.get()
+        let contractsSupportingCoInsured = contractStore.state.activeContracts
+            .filter({ $0.showEditCoInsuredInfo && $0.nbOfMissingCoInsuredWithoutTermination > 0 })
+            .compactMap({
+                InsuredPeopleConfig(contract: $0, fromInfoCard: true)
+            })
+
+        if contractsSupportingCoInsured.count > 1 {
+            editCoInsuredVm.editCoInsuredModelDetent = .init(contractsSupportingCoInsured: {
+                return contractsSupportingCoInsured
+            })
+        } else {
+            editCoInsuredVm.editCoInsuredModelFullScreen = .init(contractsSupportingCoInsured: {
+                return contractsSupportingCoInsured
+            })
+        }
+    }
+
+    static func checkForEditCoInsuredAlert(type: EditCoInsuredRedirectType) {
+        switch type {
+        case .checkForAlert:
+            Task {
+                let contractStore: ContractStore = globalPresentableStoreContainer.get()
+                await contractStore.sendAsync(.fetchContracts)
+                let missingContract = contractStore.state.activeContracts.first { contract in
+                    if contract.upcomingChangedAgreement != nil {
+                        return false
+                    } else {
+                        return contract.coInsured
+                            .first(where: { coInsured in
+                                coInsured.hasMissingInfo && contract.terminationDate == nil
+                            }) != nil
+                    }
+                }
+
+                if let missingContract {
+                    let missingContractConfig = InsuredPeopleConfig(contract: missingContract, fromInfoCard: false)
```

Comment:
@sladan-hedvig here we need to call `editCoInsuredVm.editCoInsuredModelMissingAlert = missingContractConfig`

but it's not possible since the function is static. Not sure how to solve this in a good way